kmip/tests/unit/services/server/crypto/test_engine.py

```diff
@@ -146,3 +146,123 @@ def test_create_asymmetric_key_with_invalid_length(self):
             engine.create_asymmetric_key_pair,
             *args
         )
+
+    def test_mac(self):
+        """
+        Test that MAC operation can be done with valid arguments.
+        """
+        key1 = (b'\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00'
+                b'\x00\x00\x00\x00')
+        key2 = (b'\x01\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00'
+                b'\x00\x00\x00\x00')
+        key3 = key1
```

Comment:
Go ahead and actually duplicate the key1 bytes here for key3. That way we explicitly show that the same key bytes, in completely separate objects, yield the same MAC for the same input data.